conda_env/specs/__init__.py

```diff
@@ -2,13 +2,15 @@
 from .yaml_file import YamlFileSpec
 from .notebook import NotebookSpec
 from .requirements import RequirementsSpec
+from .jinja2_file import Jinja2Spec
 from ..exceptions import SpecNotFound
 
 all_specs = [
     BinstarSpec,
     NotebookSpec,
     YamlFileSpec,
-    RequirementsSpec
+    RequirementsSpec,
+    Jinja2Spec
```

Comment:
Here is an example:

```yaml
name: test
dependencies:
- python={{ os.environ['PYTHON_VER'] }}
- six=1.10.0
```

It is not malformed yaml and will be parsed right, it will only fail at dependency resolve time.